crates/aptos/src/account/fund.rs

```diff
@@ -36,6 +36,20 @@ pub struct FundWithFaucet {
     pub(crate) profile_options: ProfileOptions,
 }
 
+impl FundWithFaucet {
+    /// Create a new instance of this command with the provided amount.
+    /// This is useful for testing.
+    pub fn default_with_amount(amount: u64) -> Self {
+        Self {
+            account: None,
```

Comment:
How does it know which account to fund if account = null? 

Can you include in your description how this works in CLI / how it will be used?

Reply:
ack; i'll add the tutorial in the next PR where the scripted transactions are implemented.

Reply:
also the faucet with this config funds the `default` account. Since this function is moved to a util under indexer-grpc/ I'll revisit there.

Reply:
There are two ways of doing this:
1. specify the profile name 2. or pass the account into this. 

Updated the code to allow the 2nd option.